README.md

```diff
@@ -17,6 +17,18 @@ But if you want to **contribute** and make this much better for other developer
 
 If you created something awesome and want to contribute then feel free to open an [pull request](https://github.com/saadpasta/developerFolio/pulls).
 
+## Table of Contents
```

Comment:
![image](https://user-images.githubusercontent.com/48270786/95018191-bdcced00-067b-11eb-9eee-174548b4c6fd.png)

This is how the index looks like right now, you've left `\` in front of the text.